packages/experimental/src/components/BreadcrumbWithOverflow/BreadcrumbWithOverflow.js

```diff
@@ -71,124 +71,166 @@ export const BreadcrumbWithOverflow = ({
-          : 0;
-        breadcrumbWidthsIncludingMargin.push(item.offsetWidth + marginWidths);
+      if (index === 0) {
+        return itemCount - 1; // the last item in the list
+      } else if (index === 1) {
+        return 1; // the first item in the list
+      } else {
+        return itemCount - index; // count down from itemCount - 2 to 1
       }
+    };
 
-      let overflowWidth = breadcrumbWidthsIncludingMargin[0];
+    if (maxVisibleBreadcrumbItems <= 1) {
+      setDisplayCount(1);
+    } else {
+      // how many will fit?
+      let willFit = 0;
+      let spaceAvailable = breadcrumbItemWithOverflow.current.offsetWidth;
+
+      if (sizingContainerRef.current) {
+        const sizingBreadcrumbItems = sizingContainerRef.current.querySelectorAll(
+          `.${carbonPrefix}--breadcrumb-item`
+        );
 
-      for (let i = 1; i < breadcrumbWidthsIncludingMargin.length; i++) {
-        // add all that will fit (ignoring overflow breadcrumb for now)
-        if (spaceAvailable >= breadcrumbWidthsIncludingMargin[i]) {
-          spaceAvailable -= breadcrumbWidthsIncludingMargin[i];
-          willFit += 1;
-        } else {
-          break;
+        const breadcrumbWidthsIncludingMargin = [];
+        for (let item of sizingBreadcrumbItems) {
+          const computedStyle = window
+            ? window.getComputedStyle(sizingBreadcrumbItems[0])
+            : null;
+          const marginWidths = computedStyle
+            ? parseFloat(computedStyle.marginLeft, 0) +
+              parseFloat(computedStyle.marginRight, 0)
+            : 0;
+          breadcrumbWidthsIncludingMargin.push(item.offsetWidth + marginWidths);
         }
-      }
 
-      // address overflow breadcrumb if needed
-      if (willFit < sizingBreadcrumbItems.length - 1) {
-        // -1 for overflow item
-        for (let i = 1; willFit > 0 && spaceAvailable < overflowWidth; i++) {
-          // Highly unlikely any useful breadcrumb-item is smaller
-          willFit -= 1; // remove one breadcrumb-item
+        let overflowWidth = breadcrumbWidthsIncludingMargin[0];
+
+        for (let i = 0; i < breadcrumbWidthsIncludingMargin.length - 1; i++) {
+          // count used one less than length to account for the included overflow item
+          const index = displayItemIndex(
+            breadcrumbWidthsIncludingMargin.length,
+            i
+          );
 
-          // we remove from 1 up not from end
-          spaceAvailable += breadcrumbWidthsIncludingMargin[i];
+          if (spaceAvailable >= breadcrumbWidthsIncludingMargin[index]) {
+            spaceAvailable -= breadcrumbWidthsIncludingMargin[index];
+            willFit += 1;
+          } else {
+            break;
+          }
+        }
+        // address overflow breadcrumb if needed
+        if (willFit > 1 && willFit < sizingBreadcrumbItems.length - 1) {
+          // -1 for overflow item
+
+          while (willFit > 1 && spaceAvailable < overflowWidth) {
```

Comment:
Actually, you don't need to test willFit > 1 here either. Might be slightly clearer to allow willFit to go back down to 0 if there really is insufficient space. Because in line 224 (at time of making this comment) we're going to set displayCount to 1 anyway if willFit ends up <1.
```suggestion
          while (spaceAvailable < overflowWidth) {
```